### DISCUSSION TITLE
有辦法直接下載到中國服檔案嗎？

### DISCUSSION THREAD

**Discussion by kesisan** [GpointChen/FFXIVChnTextPatch-GP#329] [General]
在沒有中國服客戶端的情況下，有沒有其他的渠道下載到0a0000三個檔案?

**Comment by GpointChen** [comment]
應該沒有

不過中國服客戶端下載檔案不需要登入，可以直接下載

**Comment by mokeling** [comment]
如果有需要我可以提供，[这边](https://ff14-clientdown.bluefissure.com/)也有别人做的镜像，不过是游戏全文件
